diffsims/tests/test_utils/test_sim_utils.py

```diff
@@ -132,3 +145,176 @@ def test_kinematic_simulator_invalid_illumination():
 def test_uvtw_to_uvw(uvtw, uvw):
     val = uvtw_to_uvw(uvtw)
     np.testing.assert_almost_equal(val, uvw)
+
+
+class TestHolzCalibration:
+    def test_get_holz_angle(self):
+        wavelength = 2.51 / 1000
+        lattice_parameter = 0.3905 * 2 ** 0.5
+        angle = get_holz_angle(wavelength, lattice_parameter)
+        assert approx(95.37805 / 1000) == angle
+
+    def test_scattering_angle_to_lattice_parameter(self):
+        wavelength = 2.51 / 1000
+        angle = 95.37805 / 1000
+        lattice_size = scattering_angle_to_lattice_parameter(wavelength, angle)
+        assert approx(0.55225047) == lattice_size
+
+
+class TestBetaToBst:
+    def test_zero(self):
+        data = np.zeros((100, 100))
+        bst = beta_to_bst(data, 200000)
+        assert data.shape == bst.shape
+        assert (data == 0.0).all()
+
+    def test_ones(self):
+        data = np.ones((100, 100)) * 10
+        bst = beta_to_bst(data, 200000)
+        assert data.shape == bst.shape
+        assert (data != 0.0).all()
+
+    def test_beta_to_bst_to_beta(self):
+        beta = 2e-6
+        output = bst_to_beta(beta_to_bst(beta, 200000), 200000)
+        assert beta == output
+
+    def test_known_value(self):
+        # From https://dx.doi.org/10.1016/j.ultramic.2016.03.006
+        bst = 10e-9 * 1  # 10 nm, 1 Tesla
+        av = 200000  # 200 kV
+        beta = bst_to_beta(bst, av)
+        assert approx(beta, rel=1e-4) == 6.064e-6
+
+
+class TestBstToBeta:
+    def test_zero(self):
+        data = np.zeros((100, 100))
+        beta = bst_to_beta(data, 200000)
+        assert data.shape == beta.shape
+        assert (data == 0.0).all()
+
+    def test_ones(self):
+        data = np.ones((100, 100)) * 10
+        beta = bst_to_beta(data, 200000)
+        assert data.shape == beta.shape
+        assert (data != 0.0).all()
+
+    def test_bst_to_beta_to_bst(self):
+        bst = 10e-6
+        output = beta_to_bst(bst_to_beta(bst, 200000), 200000)
+        assert bst == output
+
+
+class TestEtToBeta:
+    def test_zero(self):
+        data = np.zeros((100, 100))
+        beta = et_to_beta(data, 200000)
+        assert data.shape == beta.shape
+        assert (data == 0.0).all()
+
+    def test_ones(self):
+        data = np.ones((100, 100)) * 10
+        beta = bst_to_beta(data, 200000)
+        assert data.shape == beta.shape
+        assert (data != 0.0).all()
+
+
+class TeslaToAm:
+    def test_zero(self):
+        data = np.zeros((100, 100))
+        am = tesla_to_am(data)
+        assert data.shape == am.shape
+        assert (data == 0.00).all()
+
+    def test_ones(self):
+        data = np.ones((100, 100)) * 10
+        am = tesla_to_am(data)
+        assert data.shape == am.shape
+        assert (data != 0.0).all()
+
+    def test_known_value(self):
+        tesla = 1
+        am = tesla_to_am(tesla)
+        assert approx(tesla, rel=1e-4) == 795775
+
+
+class TestAccelerationVoltageToVelocity:
+    def test_zero(self):
```

Comment:
This isn't a physically meaningful set up (as far as I can see), so we can probably delete this test, as the one below is better and covers a lot more.